### PR TITLE
Add Gerrit change review support

### PR DESCRIPTION
**Pull request by staticmukesh**
An attempt to add Gerrit comment support: https://github.com/reviewdog/reviewdog/issues/69

### CODE REVIEW

**Comment by staticmukesh** [comment]
Hello @haya14busa, kindly review the pull request. 😄

**Comment by haya14busa** [comment]
Hi @staticmukesh, 

I'm very sorry for the late response.
It's a very exciting change! Thank you for your contribution :+1:

I can merge it, but I'm concern about the maintenance.
I don't have any gerrit repositories I manage, so I cannot test the feature and hard to maintain it.

I have 2 questions/asks.
1. @staticmukesh Are you interested in maintain the gerrit support feature? I can invite you to the org.
2. Do you know any good ways to set up a public gerrit repository including CI to run the reviewdog?

**Comment by staticmukesh** [comment]
@haya14busa Thanks for the opportunity. I am interested in maintaining the Gerrit feature.

Let me explore to set up a public Gerrit repository.

File: service/gerrit/change_diff.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	if err != nil {
+		return nil, fmt.Errorf("failed to get merge-base commit: %v", err)
+	}
+	mergeBase := strings.Trim(string(b), "\n")
+	relArg := fmt.Sprintf("--relative=%s", g.wd)
+	bytes, err := exec.Command("git", "diff", relArg, "--find-renames", mergeBase, baseSha).Output()
+	if err != nil {
+		return nil, fmt.Errorf("failed to run git diff: %v", err)
+	}
+	fmt.Println(string(bytes))
```
Comment (haya14busa) [inline service/gerrit/change_diff.go:72]:
```suggestion
```

**Comment by haya14busa** [comment]
Thanks! I invited you to the organization.

**Comment by felipecassiors** [comment]
@haya14busa by the way, check out http://gerrithub.io/.

**Comment by haya14busa** [comment]
@felipecassiors Thanks! I filed an issue to track it. #581

**Pull request merged**